components/BoardMeeting.tsx

```diff
@@ -1,14 +1,8 @@
-import Link from 'next/link'
 import TranscriptHeader from 'components/TranscriptHeader'
 import SpeakerBubble from 'components/SpeakerBubble'
-import SpeakerInfoControl from 'components/SpeakerInfoControl'
-import TranscriptControl from 'components/TranscriptControl';
-import VideoPlayer from 'components/VideoPlayer'
-import YouTube from 'react-youtube'
-import { SpeakerInfoData } from 'utilities/speaker-info'
 import type { TranscriptData } from 'utilities/transcript'
-import type { Entries } from 'type-fest';
-import { toSpeakerKey, UnknownSpeakerNum } from 'utilities/speaker-info'
+import { UnknownSpeakerNum } from 'utilities/speaker-info'
+import BoardMeetingControl from './BoardMeetingControl';
```

Comment:
nit: use the `components/BoardMeetingControl` path to allow easier relocation of this file.